Source/MonoGame.Extended/BitmapFonts/BitmapFont.cs

```diff
@@ -44,28 +44,48 @@ internal static IEnumerable<int> GetUnicodeCodePoints(string s)
 
         public Size GetSize(string text)
         {
+            var totalWidth = 0;
             var width = 0;
+            var totalHeight = 0;
             var height = 0;
+
+            const int newlineCodePoint = 10;
             var codePoints = GetUnicodeCodePoints(text).ToArray();
 
             for (int i = 0, l = codePoints.Length; i < l; i++)
             {
                 BitmapFontRegion fontRegion;
                 var c = codePoints[i];
+                var d = c;
```

Comment:
In general we should be trying to write [self documenting code](https://blog.codinghorror.com/coding-without-comments/) by naming methods and variable names clearly.

A few comments are helpful if they are truly adding *value* to the code. Just don't add comments to code where it's already obvious what's going on.

For example, adding a comment like this is completely useless and just adds clutter to the code. Please don't do this. If you feel a strong need to write comments, try renaming some variables first.

    // gets the unicode code points
    var codePoints = GetUnicodeCodePoints(text).ToArray();